lib/mix/lib/mix/tasks/deps.loadpaths.ex

```diff
@@ -16,6 +16,7 @@ defmodule Mix.Tasks.Deps.Loadpaths do
 
     * `--no-deps-check` - does not check or compile deps, only load available ones
     * `--no-compile` - does not compile dependencies
+    * `--no-prune-deps` - do not prune deps from the code path
```

Comment:
I am not sure this is the correct name or if we should simply change the `--no-deps-check` flag instead. That's because in this case, we are doing more than just skipping the pruning but we are not loading them either. There are three steps in this code:

1. We load deps
2. We unload unknown deps
3. We prune unknown deps

`--no-deps-check` already skips the 3rd step but I could see it also skipping step 2, which would probably make this patch obsolete.